asset/css/search-base.less

```diff
@@ -308,6 +321,24 @@ fieldset:disabled .term-input-area [data-drag-initiator] {
         top: 85%;
         left: .5em;
       }
+
+      .remove-action {
+        visibility: visible;
+        position: absolute;
+        width: 100%;
+        top: 0;
+        line-height: normal;
+        padding: 0.5em;
+        cursor: pointer;
```

Comment:
It currently hides the validation message in case the value is invalid. This container should not catch focus hence should use `pointer-events: none` instead of this.